common/types.go

```diff
@@ -163,8 +164,77 @@ func (ep *EvictionPolicy) Parse(s string) error {
 
 type LogConfig struct {
 	Level       LogLevel
```

Comment:
As this structure is created only once in one run, it shall be ok even if its unaligned. We need restructuring of fields when we allocate that structure in bulk.